src/components/common/YourCompany/CompanyName/CorrectNameRequest.vue

```diff
@@ -166,8 +166,12 @@ export default class CorrectNameRequest extends Mixins(CommonMixin, NameRequestM
   }
 
   private isValidNrNumber (value: string): boolean {
-    const VALID_FORMAT = new RegExp(/^(NR )\d{7}$/)
-    return VALID_FORMAT.test(value)
+    const VALID_FORMAT = new RegExp(/^(NR)?\s*(\d{7})$/)
+    if (VALID_FORMAT.test(value)) {
+      this.nameRequestNumber = 'NR ' + value.match(VALID_FORMAT)[2]
```

Comment:
What happens if `value.match(VALID_FORMAT)` is not an array with 3 or more items?

Reply:
for example?

Reply:
if the NR matches the regex, it should be an array with 3 items.

Reply:
I tried but can't find an example. Since this array indexing only happens if the test is True then I guess it's safe.